src/client/src/components/Home/ProductList/ProductItem.tsx

```diff
@@ -10,19 +10,37 @@ const mapStateToProps = ({ user, order }: { user: User; order: Order }) => {
 
 const mapDispatchToProps = (dispatch: any) => {
   return {
-    userAddToOrder: (userId: number, product: Product) => dispatch(loggedInAddToOrder(userId, product))
+    userAddToOrder: (userId: number, product: Product) =>
+      dispatch(loggedInAddToOrder(userId, product)),
+    guestAddToOrder: (product: Product) => dispatch(createOrder(product))
```

Comment:
I don't think that we should be using two different thunks for this sort of thing. I think we should hit one route and allow the server to decide which database (express-session or postgres) to return the data from.